### ISSUE TITLE
Faulty Job Name processing

### ISSUE THREAD

**Issue by DEV-Pad** [jenkinsci/discord-notifier-plugin#4]
Hi there,

I recently discovered your awesome Jenkins plugin. Good work!

However, it seems to have a bug regarding the job name processing.
Job names may contain parenthesis, but your plugins seems to cannot handle it correctly.

One of my jobs is titled "ELMRSG - Development (luckev.info) - Deployment". 
"luckev.info" is surrounded by parenthesis, so the following bug occurs:
![bug](https://user-images.githubusercontent.com/44834167/51559385-0893da80-1e82-11e9-9e3a-a9584e97b741.png)

So basically the link is broken, as the linked build number and build state refer to "https://www.luckev.info/jenkins/job/ELMRSG%20-%20Development%20(luckev.info" and the ")" disappears and the remaining characters behind ")" are simply displayed behind the actual link.

I'd be great, if you could fix this issue.
Sincerely, Pad
